test/unit/model/model-properties-test.js

```diff
@@ -0,0 +1,65 @@
+import Model from 'controller/model';
+import video from 'mock/video';
+
+const config = {
+    volume: 10,
+    mediaElement: video,
+    mediaContainer: document.createElement('div'),
+    mute: false,
+    edition: 'enterprise',
+};
+
+const item = {
+    sources: [
+        {
+            file: 'foo.mp4',
+            type: 'mp4'
+        }
+    ],
+    minDvrWindow: 120
+};
+
+const playlist = [item];
+
+describe('Transient Model Properties', function () {
+    let model;
+
+    before(function () {
+        model = new Model().setup(config);
+        model.set('playlist', playlist);
+    });
+
+    it('should have the same properties as the config', function() {
+        expect(model.get('volume')).to.eq(config.volume);
+        expect(model.get('mute')).to.eq(config.mute);
+        expect(model.get('edition')).to.eq(config.edition);
+    });
+
+    it('should properly load playlist', function() {
+        return new Promise(function (resolve, reject) {
+            model.setItemIndex(0)
```

Comment:
This is no longer a model method. See: https://github.com/jwplayer/jwplayer/pull/2520/files#diff-a181c6e20596563a55e5710b8f4dbab7R736